doc/whats-new.md

```diff
@@ -1,301 +1,297 @@
-before removing them from the database and the file system.
+The command `oq dbserver stop` and `oq workers stop` now correctly
+stops the zmq workers (relevant for the experimental zmq mode).
 
-The command `oq zip` has been extended and now works for all kind of
-calculations.
+The command `oq show` was not working in multiuser situations for
+calculations with a parent, since the parent was being read
+from the wrong directory. This issue has been fixed.
 
-The command `oq extract` has now a signature consistent with `oq export`
-and has grown the ability to connect to a remote engine server.
+The command `oq show job_info` now returns the amount of data received
+from the controller node while the computation is running: before this
+information was available only at the end of the computation.
 
-The `oq dump` command has been extended so that it is possible to extract
-a single calculation, if so desired.
+There is a new command `oq importcalc host calc_id username password`
+to import remote calculations into the local engine database. The
+commmand has some limitations: it works only for calculations without
+a parent and only if there are no clashes between the remove calculation ID
+and the local calculation ID. It should be considered an internal command.
 
-IT
----
+Internals
+--------------
 
-We have specialized packages for clusters even for Ubuntu. The
-packages meant for the controller node includes rabbitmq and celery,
-which are missing in the packages meant for the worker
-nodes. Previously, we were installing rabbitmq and celery even when
-not needed.
+A huge improvement has been made in cluster situations: now the results
+are returned via [ZeroMQ](http://zeromq.org/) and not via rabbimq.
+This allows us to bypass the limit of rabbitmq and large computations can 
+be run without running out of disk space in the mnesia directory.
+Hundreds of thousands of tasks can be generated without issues, a feat
+previously impossible.
 
-Docker containers can be used to deploy a multi-node installation of
-the Engine on a Docker cluster with dynamic scaling.
+The task distribution code has been simplified and
+code in the experimental/proof-of-concept state has been removed: in
+particular the support to ipython and the support to SGE have disappeared.
+As it is now, they were not used and they were a significant maintenance cost.
 
-It is possible to specify a `custom_tmp` variable in the openquake.cfg file
-to configure a custom path for the temporary directory. This is useful in
-situations when the system /tmp directory is in a partition which is
-too small (this happened to users with a machine in the Azure cloud).
+Now we use the port 1907 for the DbServer, when installing the engine
+from the packages. When installing from sources, the port is the number 1906,
+as before. In this way an installation from packages can coexists with
+an installation from sources out of the box.
```

Comment:
As above I would just say: `You may need to adjust the firewall configuration, if any, on a cluster master node`